reflections/shape/shape.js

```diff
@@ -46,11 +46,10 @@ try{
 function makeSerializer(methodName, symbolsToCheck){
 
 	return function serializer(value, MapType ){
-		if( typeReflections.isPrimitive(value) ) {
+		if(typeReflections.isBuiltIn(value) && !typeReflections.isPlainObject(value)) {
```

Comment:
There is a comment below that says `if it's a Date, RegExp, Map, etc ... do nothing`. Can you correct this?